packages/test-interface-criteria/README.md

```diff
@@ -21,6 +21,44 @@ const TestOpportunityBookableCriteria = TestInterfaceCriteria.criteriaMap.get('T
 const { matchesCriteria, unmetCriteriaDetails } = TestInterfaceCriteria.testMatch(TestOpportunityBookableCriteria, opportunity);
 ```
 
+## Concepts
```

Comment:
Thoughts on this section being above Usage?

Reply:
My rationale was that it seems to me to be a common pattern for usage/examples to come first in project READMEs. Perhaps geared towards people who are already somewhat knowledgeable and want to get stuck in